Server/BackEndMonitoringServer/include/CJSONFormatterProcess.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include "CJSONFormatter.h"
```

Comment:
Think about the solution hove to move this include from here

Reply:
Left only need one for inheriting